tools/pctools/README.md

```diff
@@ -0,0 +1,41 @@
+PUBLIC CLOUD TOOLS IMAGE
+========================
+
+A Public Cloud Tools image is used for specific tests for azure, amazon, openstack, ec2 and google CSPs.
```

Comment:
It is a good habit to always establish the context of the repository to allow readers without prior background information to get a grasp what's going on here, e.g.


```suggestion
The Public Cloud Tools image is a modified SLES image with a pre-installed and configured set of various cloud related tools intended for testing of Cloud Service Providers (CSP) within openQA. It include various tools including command-line interfaces for Azure, Amazon, OpenStack and Google Compute Platform (GCP). This Dockerfile helps to generate the automatically generate the requirements for those utilities.
```

It is also a courtesy to the reader to always introduce abbreviations when used for the first time (or refer to a glossary).